src/scss/components/button/_index.scss

```diff
@@ -1 +1,315 @@
-// TBD... button styles
+@import "config";
+
+/// ==================================================
+/// MIXINS - DEPRECATED! Use SCSS instead!
+/// ==================================================
+
+#Button {
+  @mixin reset() {
+    background-color: transparent;
+    border: 0;
+    color: inherit;
+    cursor: pointer;
+    display: inline-block;
+    font: inherit;
+    font-weight: 500;
+    line-height: 1;
+    margin: 0;
+    padding: 0;
+  }
+
+  @mixin basic() {
+    border-radius: 2px;
+    display: inline-flex;
+    justify-content: center;
+
+    > * + * {
+      margin-left: 0.5rem;
+    }
+
+    &:focus {
+      outline: none;
+    }
+
+    &[disabled] {
+      cursor: not-allowed;
+
+      &:focus {
+        box-shadow: none;
+      }
+    }
+  }
+
+  @mixin primary() {
+    background-color: $cyan-900;
+    color: $gray-0;
+
+    &:hover {
+      background-color: $cyan-500;
+      border-color: transparent;
+      color: $gray-0;
+    }
+
+    &:active {
+      background-color: $cyan-700;
+      border-color: transparent;
+      color: $gray-0;
+    }
+
+    &:focus {
+      box-shadow: $focus-glow;
+      outline: none;
+    }
+
+    &[disabled] {
+      background-color: $gray-400;
+      border-color: transparent;
+      color: $gray-700;
+    }
+  }
+
+  @mixin secondary() {
+    border: 1px solid $cyan-900;
+    color: $cyan-900;
+
+    &:hover {
+      background-color: $cyan-500;
+      border-color: transparent;
+      color: $gray-0;
+    }
+
+    &:active {
+      background-color: $cyan-700;
+      border-color: transparent;
+      color: $gray-0;
+    }
+
+    &:focus {
+      box-shadow: $focus-glow;
+    }
+
+    &[disabled] {
+      background-color: transparent;
+      border-color: $gray-400;
+      color: $gray-400;
+    }
+  }
+
+  @mixin tertiary() {
+    background-color: transparent;
+    border: none;
+    color: $cyan-900;
+    padding-left: 0;
+    padding-right: 0;
+
+    &:hover {
+      background-color: transparent;
+      border-color: transparent;
+      color: $cyan-500;
+    }
+
+    &:active {
+      background-color: transparent;
+      border-color: transparent;
+      color: $cyan-700;
+    }
+
+    &:focus {
+      box-shadow: $focus-glow;
+    }
+
+    &[disabled] {
+      background-color: transparent;
+      border-color: transparent;
+      color: $gray-400;
+    }
+  }
+
+  @mixin small() {
+    padding: 0.125rem 0.5rem; // 2px 8px
+  }
+
+  @mixin medium() {
+    font-size: 0.875rem;
+    padding: 8px 12px;
+  }
+
+  @mixin large() {
+    font-size: 1rem;
+    padding: 1rem;
+  }
+}
```

Comment:
Remove lines 3-142.

Button mixins have already been defined in SCSS.

Reply:
Changes have been done, please review.